### DISCUSSION TITLE
Add a thesis exposé example

### DISCUSSION THREAD

**Discussion by hochleitner** [Digital-Media/HagenbergThesis#153] [Ideas]
In our scientific writing courses in the bachelor programs, students create an exposé for their bachelor's thesis. They'll use the Hagenberg Thesis template to write their thesis, so we also have them write their exposé using it as well.

Since the bachelor's thesis is, for many, the first experience with scientific writing, we give them an exposé example document. That way, they know what's expected of them and what the structure of an exposé looks like. And while working on it, they also become familiar with the template. The document is located here: https://github.com/Digital-Media/MTD248-WA4.

@imagingbook, how do you feel about including this example in Hagenberg Thesis? It could become `HgbThesisExposeDE` (if you don't like the content, we can also come up with something else).

Including it would make it less confusing because, at the moment, there are two repos. Plus, it would get updated with the latest additions from the package automatically. Also, I think that an exposé is something that many students could benefit from.

**Comment by imagingbook** [comment]
Sure, can't hurt. Pls. add it to `documents` and update the `README` file accordingly. The build script should need no changes.

**Reply by hochleitner** [reply]
Alright, I will add this to the milestone and include it in the release for SS 2023. I created issue #155 from this discussion to track it and discuss it further.
